examples/darwin-framework-tool/commands/configuration/SetUpDeviceCommand.h

```diff
@@ -0,0 +1,82 @@
+/*
+ *   Copyright (c) 2024 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+#pragma once
+
+#import "../common/DeviceDelegate.h"
+#import <Matter/Matter.h>
+
+#include <commands/common/Command.h>
+
+class SetUpDeviceCommand : public CHIPCommandBridge {
+public:
+    SetUpDeviceCommand()
+        : CHIPCommandBridge("device", "Creates and configures an instance of a device.")
+    {
+        AddArgument("node-id", 0, UINT64_MAX, &mNodeId, "The Node ID of the device instance to create.");
+        AddArgument("pretend-thread-enabled", 0, 1, &mPretendThreadEnabled,
+            "When the device is configured using an MTRDevice (via -use-mtr-device), instructs the MTRDevice to treat the "
```

Comment:
I don't get this comment. This command _always_ sets up an MTRDevice...  What device is "configured" with "-use-mtr-device" for purposes of this command?  (And shouldn't it be --use-mtr-device anyway?)